src/calendar/__tests__/month-granularity.test.tsx

```diff
@@ -0,0 +1,99 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { render } from '@testing-library/react';
+import styles from '../../../lib/components/calendar/styles.selectors.js';
+import screenreaderOnlyStyles from '../../../lib/components/internal/components/screenreader-only/styles.selectors.js';
+import Calendar, { CalendarProps } from '../../../lib/components/calendar';
+import createWrapper from '../../../lib/components/test-utils/dom';
+
+import * as React from 'react';
+
+const defaultProps: CalendarProps = {
+  granularity: 'month',
+  i18nStrings: {
+    currentMonthAriaLabel: 'Current month',
+    nextYearAriaLabel: 'Next year',
+    previousYearAriaLabel: 'Previous year',
```

Comment:
I think we should test the same labels also with i18n provider.

Reply:
I don't see we do this with any other component. Should this become a general rule then?

Reply:
We do have it in other components but the coverage could have been better, see: https://github.com/search?q=repo%3Acloudscape-design%2Fcomponents%20TestI18nProvider&type=code

Reply:
Oh I see we do it with this utility. I'll take a look.

Reply:
Added coverage [here](https://github.com/cloudscape-design/components/pull/1942/commits/11e1d2c472df36f532fdf279076fbdd4a140866d).